### ISSUE TITLE
Allow personalized commit messages on update cli command

### ISSUE THREAD

**Issue by ricoms** [mbarkhau/bumpver#162]
I propose user can define commit message on command line interface when running bumpver update:

goal is to have something like below:
```
poetry run bumpver update -f -t final --commit-message "[final-version] {old_version} -> {new_version}"

**Comment by mbarkhau** [comment]
I'm guessing you are aware of this, I just want to be sure: There is the [`commit_message` configuration option](https://github.com/mbarkhau/bumpver#vcs-parameters-gitmercurial).

Is the idea here, to be able to override the configured option (or the default) via the cli?

**Comment by ricoms** [comment]
Yes. I'm aware of that, my use-case is that I want to provide different commit messages on different stages of versioning. I pretend to use those commit messages on CI/CD pipelines.

**Comment by mbarkhau** [comment]
Doc update: https://github.com/mbarkhau/bumpver#custom-commit-message

Released with [`bumpver==2021.1113`](http://pypi.python.org/pypi/bumpver/2021.1113/).

Cheers
